### ISSUE TITLE
[ACCESSIBILITY] The Maps feature is inaccessible to users of assistive technology

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#3025]
**Describe the bug**
The Maps feature is inaccessible to users of assistive technology. The map itself does not allow the screen reader user to gesture or select 

**To Reproduce
Steps to reproduce the behavior:**
4.	Open app, logged in as student
5.	Turn Talkback on
6.	Gesture to MTD Destinations in the Maps module
7.	Close the modal that appears with message “Please tap a location…”
8.	Select a location like “Mumford Hall” 

Expected behavior
****This should behave for AT users the same way it does for users of assistive technology except AT users will be hearing the labels and buttons as they gesture through instead of reading only. 

**Screenshots**

https://user-images.githubusercontent.com/82892873/224029978-a29d90e4-6ede-4d04-bb35-d1fbbf9a88af.mp4

https://user-images.githubusercontent.com/82892873/224029995-1f9653e4-ee6d-40f4-b1d7-c69250efe1dd.mp4

**Testing Environment & Devices (please complete the following information):**
•	OS: Android Version 13
•	Device: Google Pixel 4a

**Comment by TodorBachvarov** [comment]
@acovher2 The Google Maps seems to work in a similar way so this is the default Android Accessibility Map behaviour. Nothing that we can do to improve.

**Comment by acovher2** [comment]
@TodorBachvarov @pmarkhennessy I do not agree that there is nothing to be done.  
Please read this link: http://web-accessibility.carnegiemuseums.org/content/maps/

A clip from the sight that pertains to this ticket is below:
"Google Maps
An embedded Google Map alone is not screen reader accessible. When using Google Maps, follow these tips:

Descriptive heading: Include a heading right before the map iFrame which describes the purpose of the map.
Determine the purpose of the map: If the map is on the page to show landmarks, describe those landmarks and their locations, either in an unordered list or a paragraph. If the map is to show directions, then those directions should be spelled out in basic, text-based directions.
Title attribute: A Google Map is typically embedded within an iFrame. Make sure the iFrame includes a title attribute.
Embedded Google Map Example:
In the example below, we’ve embedded a Google Map, and we’re setting it off with a descriptive heading which is also linked to the Google Maps site. A screen reader should announce that the heading is a link, and if the user chooses, they will be taken to the Google Maps site. On mobile devices with the Google Maps app installed, users should be automatically prompted as to whether or not they’d like to open the link in their Google Maps app. Under the heading, we have provided a description of where the Carnegie Musuems are, as that is the landmark we are highlighting with the Google Maps iFrame.

Based on our research, users relying on screen reader software use the Google Maps mobile app frequently, and find it easy to use.

Live Example
[Carnegie Museum & Library Location](https://www.google.com/maps?ll=40.443325,-79.950485&z=16&t=m&hl=en-US&gl=US&mapclient=embed&cid=4136382053310121540):
The Carnegie Museums of Pittsburgh are located at 4400 Forbes Avenue in the Oakland neighborhood of Pittsburgh, PA and shares the same building space as the Carnegie Music Hall and the Carnegie Library of Pittsburgh Oakland branch."

**Comment by acovher2** [comment]
@TodorBachvarov Regarding "@acovher2 The Google Maps seems to work in a similar way so this is the default Android Accessibility Map behaviour. Nothing that we can do to improve." 

Do you mean to say we cannot change the Tab order of things within the application as relates to Maps?
1 - Items are hidden behind the Building overlay at the bottom of the Map. 
2 - When user selects a given building, the expectation would be that the focus for screen reader would be taken to the overlay with the details and "Directions" and "Details" CTA's. 
Please watch and advise. 

https://github.com/user-attachments/assets/2ccaacac-498b-4d96-8fd0-e3fa42b87707

Due to the ADA title II changes and deadline of April 2026, I wanted to ensure this gets on the list and radar as we need to be compliant.
